raiden/ui/app.py

```diff
@@ -138,6 +138,7 @@ def run_app(
     routing_mode: RoutingMode,
     config: Dict[str, Any],
     grpc_endpoint: str,
+    grpc_client_timeout: float,
```

Comment:
For testing you will need to pass a float number to the client, for example 0.1 seconds to make it fail by timeout to check if the timeout is ok, i agree on the default values but the client should accept floats as the parameter.

Reply:
1. i will keep it like it is defined on grpc, a float. I know that nobody will use 10.1.. but our test cases probably will use 0.1 or something to force failures
2. it should be a float then

Reply:
we agreed to keep this as a float because it's already going to be used as such in the client, plus it's not such a hindrance on the user to have this option besides the integer